addons/mail/static/src/js/utils.js

```diff
@@ -43,6 +43,11 @@ var urlRegexp = /\b(?:https?:\/\/\d{1,3}(?:\.\d{1,3}){3}|(?:https?:\/\/|(?:www\.
  * @return {string} linkified text
  */
 function linkify(text, attrs) {
+    function escapeEntities (str) {
```

Comment:
In what way does that behave differently than `_.escape`?

Reply:
escape from `_` (underscord lib) is escaping all tags characters and all attribute characters (in particular `'`). But `'` should not be escaped when injecting anywhere else than inside an attribute that has been opened with `'`.

Reply:
it was to avoid messing with characters like ' that are used in smileys (so we would need to fix smiley handling too) and preventing `linkify` from modifying the content when there is no link (it's not an issue since `"` or `&quot;` in html should be equivalent, but not useful)

`_.escape` escapes `&`, `<`, `>`, `"`, `\``, `'`

in text nodes, the browser doesn't escape " ' and `

Reply:
OK, that probably deserves a clear explanation in the commit message, or a comment on the function.